tests/v2/test_jsonrpc_waku.nim

```diff
@@ -51,7 +52,7 @@ procSuite "Waku v2 JSON-RPC API":
 
     # RPC server setup
     let
-      rpcPort = Port(8545)
+      rpcPort = Port(8546)
```

Comment:
Is there a better way we can handle getting an open port?

Reply:
What do you mean? In this PR, I have just put random numbers per test to avoid chained failures.

I guess what you are referring to is to allow putting a `Port(0)` and getting a random TCP port. This is work that needs to be done. Feel free to jump on that task and do it :)

Reply:
perhaps @rymnc refers to a getRandomPort() function that we can use instead of manually setting a random port? this function could use a hardcoded seed so that ports are generated deterministically, but randomly.